### PR TITLE
docs: reference architecture updates

### PR DESCRIPTION
**Pull request by Beilmann**
### Description

Replace this text with a summary of the changes in this PR. Include why the changes are needed and context about the changes. List required dependencies. If there is a Git issue for the change, please link to it.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

File: reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+{: caption="Table 2. PowerVS workspace architecture decisions" caption-side="bottom"}
+
+### Key and password management architecture decisions
+{: #full-key-pw}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* Use public/private SSH key to access virtual server instances by using SSH  \n * Use SSH proxy to log in to all virtual server instances by using the bastion host  \n * Do not store private ssh key on any virtual instances, also not on the bastion host  \n * Do not allow any other SSH login methods except the one with specified private/public SSH key pair|Public SSH key - provided by customer. Private SSH key - provided by customer.|Ask customer to specify the keys. Accept the input as secure parameter or as reference to the key stored in IBM Cloud Secure Storage Manager. Do not print SSH keys in any log files. Do not persist private SSH key.|                    |
```
Comment (SirSpidey) [inline reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md:103]:
```suggestion
|* Use public/private SSH key to access virtual server instances by using SSH  \n * Use SSH proxy to log in to all virtual server instances by using the bastion host  \n * Do not store private SSH keys on any virtual instances or on the bastion host  \n * Do not allow any other SSH login methods except the one with specified private and public SSH key pairs|Public SSH key - provided by customer. Private SSH key - provided by customer.|Ask customer to specify the keys. Accept the input as secure parameter or as reference to the key stored in IBM Cloud Secure Storage Manager. Do not print SSH keys in any log files. Do not persist private SSH key.|                    |
```

File: reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+{: caption="Table 3. Key and passwords management architecture decisions" caption-side="bottom"}
+
+## Compliance
+{: #compliance}
+
+This reference architecture is certified for SAP deployments.
```
Comment (SirSpidey) [inline reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md:109]: Is the reference architecture or the deployable architecture certified?

File: reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Next steps
+{: #next-steps}
+
+As next step, you may perform SAP system installation.
```
Comment (SirSpidey) [inline reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md:114]:
```suggestion
Install the SAP system.

```

File: reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md (hunk 1 above)
Comment (SirSpidey) [reply reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md:103]: @Beilmann I just noticed the reference to `IBM Cloud Secure Storage Manager`. I can't find that name anywhere in IBM Cloud. Is that a cloud service or a generic name for storing your key securely. 

If it's the 2nd, would this work?
> Accept the input by secure parameter or by reference to the key stored in an IBM Cloud key management system.

**Review by surajsbharadwaj — APPROVED** [review]

**Pull request merged**